rpmlint/descriptions/CheckForXinetd.toml

```diff
@@ -0,0 +1,3 @@
+obsolete-xinetd-requirement="""
+In systemd based distributions xinetd has become obsolete.
```

Comment:
Could you please mention socket activated services here?

Reply:
Sure, please provide a wording? I'm not so much familiar with `systemd`.

Reply:
Something like:

Xinetd is obsolete by systemd socket activated services.
Please stop using xinetd and switch to socket activation
from systemd.

Reply:
Thanks, done.